src/simulations/modeling/operations.py

```diff
@@ -35,15 +35,30 @@ def apply_operations(model, operations):
                              f"'{operation['type']}'")
 
 
+def _parse_metabolite(metabolite_id):
+    """Create a metabolite object from an identifier."""
+    try:
+        compartment = metabolite_id.rsplit("_", 1)[1]
```

Comment:
Perhaps we should also check that `compartment in model.compartments` is true here?

Reply:
Yes, I like the code that you wrote in your PR. Maybe we can use the same utility function?

Reply:
That makes sense. I'll keep this in mind and create a refactoring PR soon.